dot/state/block_notify.go

```diff
@@ -62,19 +71,21 @@ func (bs *BlockState) RegisterFinalizedChannel(ch chan<- *types.FinalisationInfo
 	return id, nil
 }
 
-// UnregisterImportedChannel removes the block import notification channel with the given ID.
-// A channel must be unregistered before closing it.
-func (bs *BlockState) UnregisterImportedChannel(id byte) {
+// FreeImportedBlockNotifierChannel to free and close imported block notifier channel
```

Comment:
where is the channel closed?

Reply:
It's not being closed, I've updated comment.